client/app/src/accounts/account.controller.js

```diff
@@ -600,7 +600,7 @@
     };
 
     self.myAccountsBalance = function() {
-      return (self.getAllAccounts().reduce(function(memo, acc) {
+      return (self.myAccounts().reduce(function(memo, acc) {
```

Comment:
Just a thought. Maybe change https://github.com/liamreoch/ark-desktop/blob/571d1a0c585f09eb3bd77c1bcbdcd8468711e337/client/app/src/accounts/account.controller.js#L586 to use `self.myAccounts()` instead of `self.accounts` (instead of changing this line). The reason being is that this will stop the balance for ledger wallets. Maybe give it a try 🙂